htdocs/xoops_lib/modules/protector/library/HTMLPurifier/HTMLModule/TargetNoreferrer.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/**
+ * Module adds the target-based noreferrer attribute transformation to a tags.  It
+ * is enabled by HTML.TargetNoreferrer
+ */
+class HTMLPurifier_HTMLModule_TargetNoreferrer extends HTMLPurifier_HTMLModule
```

Comment:
__[code_sniffer]__
* Each class must be in a namespace of at least one level (a top-level vendor name)
* Class name "HTMLPurifier_HTMLModule_TargetNoreferrer" is not in camel caps format

<!-- Comment by [SideCI](https://sideci.com) -->